buddyboss/buddyboss-platform
# Forum search issue - fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2069  .

### How to test the changes in this Pull Request:

1. Go to Forums page
2. Search for a forum that is connected with a group
3. Also make sure to Group checkbox is enabled from BuddyBoss > Settings > Search page
4. See the ajax search is working properly

### Proof Screenshots or Video

https://www.loom.com/share/2ce5dfec9a034ccc89f362378355440a

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
